src/mrinufft/trajectories/display3D.py

```diff
@@ -0,0 +1,122 @@
+"""Utils for displaying 3D trajectories."""
+
+from mrinufft import get_operator, get_density
+from mrinufft.trajectories.utils import (
+    convert_trajectory_to_gradients,
+    convert_gradients_to_slew_rates,
+    KMAX,
+    DEFAULT_RASTER_TIME,
+)
+import numpy as np
+from typing import Tuple
+from mrinufft.io import read_trajectory
+
+
+def get_gridded_trajectory(
+    shots: np.ndarray,
+    shape: Tuple,
+    grid_type: str = "density",
+    osf: int = 1,
+    backend: str = "gpunufft",
+    traj_params: dict = None,
+    turbo_factor: int = 176,
+    elliptical_samp: bool = True,
+    threshold: float = 1e-3,
+):
```

Comment:
Overall, why have all of those features gathered into only one function ? Some of them are not related to grids.

Reply:
Also please add more comments to your code

Reply:
I think, all the features are related to gridding and how you want to "color" the trajectories on the grid.